release/mgp/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "mgp"
+version = "1.0.0"
+description = "Memgraph's module for developing MAGE modules. Used only for type hinting!"
+authors = [
+    "MasterMedo <mislav.vuletic@gmail.com>",
```

Comment:
Is it wanted?

Reply:
I have no preference but I would leave people who have made some contributions, unless we want otherwise

Reply:
Me neither, it was simply to check it was not a "wrong" copy paste :)